modules/fs/fs.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2019 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package fs
+
+import "github.com/spf13/afero"
+
+// AppFs is an abstract filesystem layer
```

Comment:
I think we can remove this file to just use `os`.

Reply:
FIxed